docs/stake-pool-guide/node-installation-process.md

```diff
@@ -0,0 +1,239 @@
+---
+id: node-installation-process
+title: Cardano Node Installation process (Environment Setup)
+sidebar_label: Cardano Node Installation process
+description: Cardano Node Installation process (Environment Setup)
+image: ../img/og-developer-portal.png
+---
+The Cardano node is the core component that underpins the Cardano network. Ultimately, a blockchain network is just a collection of interconnected nodes, all working together to validate transactions and blocks by means of consensus. The definition of consensus for any given network varies, but for the Cardano network it’s defined by the [Ouroboros](stake-pool-basics#understanding-consensus) protocol.
+
+In this section, we will walk you through the process of downloading, compiling, and installing `cardano-node` and `cardano-cli` into your **Linux-based** operating system.
+
+## Installing Operating System dependencies
+
+To download the source code and build it, you need the following packages and tools on your Linux system:
+
+* the version control system `git`,
+* the `gcc` C-compiler,
+* C++ support for `gcc`,
+* developer libraries for the arbitrary precision library `gmp`,
+* developer libraries for the compression library `zlib`,
+* developer libraries for `systemd`,
+* developer libraries for `ncurses`,
+* `ncurses` compatibility libraries,
+* the Haskell build tool `cabal`,
+* the GHC Haskell compiler (version `8.10.7` or above).
+
+In Redhat, Fedora, and Centos:
+```bash
+sudo yum update -y
+sudo yum install git gcc gcc-c++ tmux gmp-devel make tar xz wget zlib-devel libtool autoconf jq -y
+sudo yum install systemd-devel ncurses-devel ncurses-compat-libs -y
+```
+
+For Debian/Ubuntu, use the following instead:
+```bash
+sudo apt-get update -y && sudo apt-get upgrade -y
+sudo apt-get install automake build-essential curl pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf -y
+```
+If you are using a different flavor of Linux, you will need to use the correct package manager for your platform instead of `yum` or `apt-get`, and the names of the packages you need to install might differ.
+
+## Installing GHC and Cabal
+
+The fastest way to install **GHC** (Glasgow Haskell Compiler) and **Cabal** (Common Architecture for Building Applications and Libraries) is to use [ghcup](https://www.haskell.org/ghcup).
```

Comment:
```suggestion
A popular way to install **GHC** (Glasgow Haskell Compiler) and **Cabal** (Common Architecture for Building Applications and Libraries) is to use [ghcup](https://www.haskell.org/ghcup).
```
`ghcup` cannot be the "fastest" way to install GHC because the other way involves unpacking a tarball & running `make install` ... which `ghcup` with its release management also must do.